src/main/common/components/payment-intention/payment-option.ts

```diff
@@ -24,6 +26,11 @@ export abstract class AbstractPaymentOptionPage<Draft> {
     return 'components/payment-intention/payment-option'
   }
 
+  async saveDraft (locals: { user: User, draft: DraftWrapper<Draft>,claim: Claim }): Promise<void> {
```

Comment:
passing claim, but not used?

Reply:
it is used here `PaymentOptionPage.generateCourtCalculatedPaymentIntention(locals.draft.document, locals.claim, decisionType)`